src/pages/setpassword/SetPasswordPageNarrow.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+    SafeAreaView,
+    View,
+    Image,
+} from 'react-native';
+import withOnyx from 'react-native-onyx/lib/withOnyx';
+import styles from '../../styles/styles';
+import SignInPageLayout from '../signin/SignInPageLayout';
+import WelcomeText from '../../components/WelcomeText';
+import ONYXKEYS from '../../ONYXKEYS';
+import SetPasswordForm from './SetPasswordForm';
+import welcomeScreenshot from '../../../assets/images/welcome-screenshot-wide.png';
+import setPasswordPagePropTypes from './setPasswordPagePropTypes';
+
+const SetPasswordPageNarrow = props => (
```

Comment:
Not sure I will block on this change, but I don't think the code we have now is the correct pattern to use. Rather than blindly follow examples that already exist we should try to understand _why_ they exist and improve them.

Reply:
Mainly the idea that we need a Narrow and a Wide version of things... there is no clear reason why this can't exist in a single file that is using the screen width to present different views. It shares many of the same elements.